src/main/java/dev/opentrading/core/spring/CoreApplicationRunner.java

```diff
@@ -39,18 +43,36 @@ public void run(ApplicationArguments args) {
                     coreBootstrap.getTradingInstruments().thenAccept(tradingInstrument -> {
                         log.info("Filtered trading instruments");
 
-                        coreBootstrap.initWebSocket().thenAccept(initWebsocket -> {
+                        coreBootstrap.<Boolean> initWebSocket().thenApply(initialized -> {
                             log.info("Initialized WebSocket");
 
-                            coreBootstrap.streamTicks().thenAccept(ScheduledStreamingTicks -> {
-                                log.info("Started streaming ticks");
-
-                                coreBootstrap.scheduleTicksAggregator().thenAccept(success -> {
-                                    log.info("Scheduled tick aggregator");
-
-                                    log.info("Started CoreApplicationRunner.");
-                                });
-                            });
+                            List<CompletableFuture<Boolean>> completableFutureList = new ArrayList<>();
```

Comment:
We’re not doing any tick aggregation right now, right? Can we delete these, and start the scheduler here?